src/main/java/hudson/remoting/jnlp/Main.java

```diff
@@ -300,6 +307,8 @@ public Engine createEngine() {
                 headlessMode ? new CuiListener() : new GuiListener(),
                 urls, args.get(0), agentName, directConnection, instanceIdentity, new HashSet<>(protocols));
         engine.setWebSocket(webSocket);
+        if(webSocketHeaders!=null)
+            engine.setWebSocketHeaders(webSocketHeaders);
```

Comment:
I do not suppose we need to imitate KK’s unusual formatting in new code, but does not matter!

Reply:
I tend to favour consistency over correctness. Given there are a bunch of similar lines above/below it, I'd preder to keep that block consistent (either mimicking KK's style or replacing it all)

Reply:
We can refactor it in a follow-up if anyone has time

Reply:
We've been trying to move Remoting code to follow a more standard formatting style. But, this particular area hasn't been updated yet, so I suppose it's okay to follow the local formatting.